pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -51,6 +52,9 @@ const (
 	backoffExponentDefault = 1.5
 	backoffDurationDefault = 5 // in seconds
 	backoffJitterDefault   = 1.0
+
+	scaleModeVMSS = "vmss"
```

Comment:
I would use something like:

`vmTypeVMSS`
`vmTypeStandard`

or something like that.